src/experimentalConfig.js

```diff
@@ -6,3 +6,5 @@
 
 export const discoverSheetAvailable = false;
 export const chartExpandedAvailable = false;
+export const nativeTransactionListAvailable = true;
```

Comment:
Btw, shouldn't this be merged with `false` in the first place?

Reply:
hm, right